web/flat-web/src/pages/LoginPage/index.tsx

```diff
@@ -67,8 +67,10 @@ export const LoginPage = observer(function LoginPage() {
             const doLogin = (loginChannel: LoginButtonProviderType): void => {
                 switch (loginChannel) {
                     case "agora": {
-                        loginDisposer.current = agoraLogin(async authData => {
-                            globalStore.updateUserInfo(authData);
+                        loginDisposer.current = agoraLogin(async ({ agoraSSOLoginID, ...rest }) => {
+                            globalStore.agoraSSOLoginID = agoraSSOLoginID ?? null;
```

Comment:
Mutating MobX properties outside of actions is not permitted. Make a `globalStore.updateAgoraSSOLoginID`.